test/uk/gov/hmrc/plasticpackagingtax/registration/views/organisation/PartnershipNameViewSpec.scala

```diff
@@ -29,7 +29,7 @@ import uk.gov.hmrc.plasticpackagingtax.registration.views.tags.ViewTest
 @ViewTest
 class PartnershipNameViewSpec extends UnitViewSpec with Matchers {
 
-  private val page = instanceOf[partnership_name]
```

Comment:
yeah; this diff looks like an improvement; inject replacing a suspicious instanceOf